ra2ce/analyses/direct/analyses_direct.py

```diff
@@ -141,17 +152,17 @@ def road_damage(self, analysis: dict) -> gpd.GeoDataFrame:
 
             return event_gdf.gdf
 
-        elif analysis["event_type"] == "return_period":
+        elif analysis.event_type == "return_period":
             return_period_gdf = DamageNetworkReturnPeriods(road_gdf, val_cols)
             return_period_gdf.main(
                 damage_function=damage_function,
                 manual_damage_functions=manual_damage_functions,
             )
 
             if "risk_calculation" in analysis:  # Check if risk_calculation is demanded
```

Comment:
This is no longer a dictionary, therefore it should be checked

`if analysis.risk_calculation:`

Reply:
Agree